www/assets/js/lib/constants.ts

```diff
@@ -118,7 +119,9 @@ const RESOURCE_TYPES = [
   "Recitation Videos",
   "Recitations",
   "Simulation Videos",
+  "Editable files",
```

Comment:
Should this be "Editable Files" and  "Student Work" to match all the other resource types? I guess you will need to edit the tag in ocw-studio as well

Reply:
Thanks for catching this @abeglova. I agree, these should have initial caps for consistency. 

```suggestion
  "Editable Files",
```

And we do need to change this in ocw-studio as well. Hopefully we don't have too much data yet. 😳